### PR TITLE
fix(quickSearch): 617 highlight in pane

### PR DESCRIPTION
**Pull request by cy**
Closes #617 

highlights all fields in the right quicksearch pane that appears in api's 'quick_search_fields'

### CODE REVIEW

File: app/scripts/components/quick-search/templates/participant.html [hunk 1]
```diff
@@ -3,11 +3,13 @@
     <i class="fa fa-user fa-12x"></i>
   </div>
 </div>
+<div>
+</div>
```
Comment (alex-wilmer) [inline app/scripts/components/quick-search/templates/participant.html]: What's this for?

Comment (cy) [inline app/scripts/components/quick-search/templates/participant.html]: my bad ✂️it

**Comment by alex-wilmer** [comment]
👍

**Comment by shanewilson** [comment]
👍

**Pull request merged**